configs/model_args_bardi.yml

```diff
@@ -0,0 +1,88 @@
+save_name: 'model'  # foldX.h5 will be appended at the end; leave blank to autopopulate with a timestamp
+task_unks:
+   task_1: 'dark_side_dx'
```

Comment:
my understanding of this is that it should be
'dark_side_dx' : "2"
Name of task is 'dark_side_dx' if we see value that we have not seen before assign "2" to it since the regular mapping is to "0" and "1" so "2" would be for unknown

Reply:
Fixed